eng/pipelines/templates/jobs/ci.yml

```diff
@@ -50,15 +50,11 @@ jobs:
           ArtifactName: 'packages'
 
     steps:
-      - ${{if eq(parameters.TestPipeline, 'true')}}:
-        - task: PowerShell@2
-          displayName: Prep template pipeline for release
-          condition: succeeded()
-          inputs:
-            pwsh: true
-            workingDirectory: $(Build.SourcesDirectory)
-            filePath: eng/scripts/SetTestPipelineVersion.ps1
-            arguments: '-BuildID $(Build.BuildId)'
+      - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+        parameters:
+          PackageName: "azure-sdk-template"
+          ServiceDirectory: "template"
+          TestPipeline: ${{ parameters.TestPipeline }}
 
       - script: |
           echo "##vso[build.addbuildtag]Scheduled"
```

Comment:
What about the other case? https://github.com/Azure/azure-sdk-for-java/commit/0d95698d3179cff99e96b4a5a7154231ca387ae3#diff-78f6357ab2cf93b8a11c1fb81a2ba78da1e364b1d14be096d9076e0ccfb2b324R186 Do you not need to add that to the Analyze job?

Reply:
Right.